### DISCUSSION TITLE
How to Forecast with Multiple features with Neural Prophet...

### DISCUSSION THREAD

**Discussion by ahsan-shabbir** [ourownstory/neural_prophet#512] [General]
I want to predict with multiple features I was thinking adding the regressor method of prophet will work with it that's not working any other way we can integrate multiple features for prediction Thanks.

**Comment by Cpt-Falcon** [comment]
Bump

**Comment by ourownstory** [comment]
Are you looking to forecast multiple series or using multiple regressors?
Both questions have been answered here - try using search :)

**Reply by Cpt-Falcon** [reply]
@ourownstory Can you explain the difference between multiple series or multiple regressors?

**Reply by ourownstory** [reply]
As in, are you trying to forecast the future of multiple series?
Or are you predicting one series, but have multiple series to regress on?
